src/dev-spaces-preview/azext_dev_spaces_preview/custom.py

```diff
@@ -38,33 +38,35 @@ def ads_use_dev_spaces(cluster_name, resource_group_name, space_name='default',
 
     from subprocess import PIPE
     should_create_resource = False
+    create_resource_ret_code = 0
     retCode = subprocess.call(
         [azds_cli, 'resource', 'select', '-n', cluster_name, '-g', resource_group_name],
         stderr=PIPE)
     if retCode == 1:
```

Comment:
Can other codes occur besides 0/1?

Reply:
Yes there could be some other return code beside 1/0. 
create_resource_ret_code = subprocess.call()

So we have a check of, if create_resource_ret_code == 0:
in case it is anything besides 0 we won't proceed